tests/unit/insertTestResult.unitTest.ts

```diff
@@ -1389,22 +1389,21 @@ describe("insertTestResult", () => {
 
             return testResultsService.insertTestResult(testResultWithAdrTestTypeWithoutExpiryDate)
                 .then((data: any) => {
-                    console.log("THIS IS THE DATA", data);
                     expect.fail();
                 })
                 .catch((error: { statusCode: any; body: any; }) => {
-                    console.log("THIS IS THE ERROR", error);
                     expect(error).to.be.instanceOf(HTTPError);
                     expect(error.statusCode).to.be.eql(400);
                     expect(error.body).to.be.eql("Expiry date not present on ADR test type");
                 });
         });
     });
 
-    context("when inserting a testResult that has an ADR testType without a certificateNumber", () => {
-        it("should throw 400 and descriptive error message", () => {
-            const testResultWithAdrTestTypeWithoutExpiryDate = testResultsPostMock[6];
-            delete testResultWithAdrTestTypeWithoutExpiryDate.testTypes[0].certificateNumber;
+    context("when inserting a testResult that has a fail ADR testType without expiryDate", () => {
+        it("should not throw error", () => {
+            const testResultWithAdrTestTypeWithoutExpiryDate = JSON.parse(JSON.stringify(testResultsPostMock[6]));
```

Comment:
Better to use lodash or similar to do a clean cloneDeep. JSON.parse(JSON.stringify(X)) can have unintended side effects.